imports/plugins/core/orders/server/no-meteor/schemas/schema.graphql

```diff
@@ -401,6 +406,30 @@ input PlaceOrderInput {
   payments: [PaymentInput]
 }
 
+"Input for the updateOrder mutation"
+input UpdateOrderInput {
```

Comment:
Does `customFields` need to be added in the mutation here, or is that going to ben an extension in its' own package?

Reply:
It will need to be added if used because we don't know what the schema should be.